packages/vite/src/node/utils.ts

```diff
@@ -464,3 +464,16 @@ export function combineSourcemaps(
 export function unique<T>(arr: T[]): T[] {
   return Array.from(new Set(arr))
 }
+
+export function defaultHostname(host: string | undefined) {
```

Comment:
**suggestion (dx, readability):** Isn't it more like a normalize or fallback?

Reply:
It is hard to name, it isnt a normalization or fallback either. More like useLocalhostIfPossible

Reply:
Ah I see, I trapped and thought it would be the other way around  

`useLocalhostIfPossible` is ugly, but definitely more expressive what happens 🤔

Reply:
Ya, in https://github.com/vitejs/vite/pull/3383 this function is gone so we can discuss how to name other things there :)